src/Apps/Order2/Routes/Checkout/Components/Order2CollapsibleOrderSummary.tsx

```diff
@@ -0,0 +1,222 @@
+import ChevronDownIcon from "@artsy/icons/ChevronDownIcon"
+import { Box, Clickable, Flex, Image, Spacer, Text } from "@artsy/palette"
+import { RouterLink } from "System/Components/RouterLink"
+import createLogger from "Utils/logger"
+import type { Order2CollapsibleOrderSummary_order$key } from "__generated__/Order2CollapsibleOrderSummary_order.graphql"
+import type * as React from "react"
+import { useEffect, useState } from "react"
+import { graphql, useFragment } from "react-relay"
+
+const logger = createLogger("Order2CollapsibleOrderSummary")
+
+interface Order2CollapsibleOrderSummaryProps {
+  order: Order2CollapsibleOrderSummary_order$key
+}
+
+export const Order2CollapsibleOrderSummary: React.FC<
+  Order2CollapsibleOrderSummaryProps
+> = ({ order }) => {
+  const orderData = useFragment(FRAGMENT, order)
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  const artwork = orderData.lineItems[0]?.artwork
+  const artworkVersion = orderData.lineItems[0]?.artworkVersion
+
+  const requiredArtworkDataPresent =
```

Comment:
`isRequiredArtworkDataPresent`

(in JS, booleans for the most part should always use that `isFoo`)